content/concepts/nat/autonat.md

```diff
@@ -10,17 +10,26 @@ aliases:
 ## Background
 
 While the [identify protocol][spec_identify] lets peers inform each other about their observed network
-addresses, however, it is possible that some of these addresses are not accessible from outside the network, as the peer may be located in a private network that is behind a [NAT](overview.md) and therefore unreachable.
+addresses, however, it is possible that some of these addresses are not accessible from outside the network,
+as the peer may be located in a private network that is behind a [NAT](overview.md) and therefore unreachable.
 
-To prevent this problem of advertising undialable addresses, libp2p has implemented a protocol called AutoNAT, which allows nodes to determine
-whether or not they are behind a NAT.
-Furthermore, this lays the foundation for nodes to find a way to improve their connectivity to peers in public networks.
+To allow peers to determine their position on the network and act accordingly, libp2p has implemented a protocol
+called AutoNAT.
 
 ## What is AutoNAT?
 
-AutoNAT allows a node to request other peers to dial its presumed public addresses. If a few of these
-dial attempts are successful, the node can be reasonably ascertain that it is not behind a NAT. On the other
-hand, if a few of these dial attempts fail, it strongly indicates that a NAT is blocking incoming connections.
+AutoNAT allows a node to request other peers to dial its presumed public addresses. If most of these dial
+attempts are successful, the node can be reasonably ascertain that it is not behind a NAT. On the other hand,
+if most of these dial attempts fail, it strongly indicates that a NAT is blocking incoming connections.
+
+Nodes can then take appropriate action based on that information. For example, if a node determines that it
+is behind a NAT, it may switch to DHT server mode or try to get a reservation with a relay in order to improve
+its connectivity to public networks.
```

Comment:
Can we move this to the first section? Also, we should have a list of what happens here.
Private:
* don't advertise private addresses
* get a reservation with a relay (link to document), and advertise relay addresses
Public: 
* start a relay to help out other nodes

Then you can mention that application protocols can also adjust their behavior. DHT Server mode is activated for public nodes, not for private nodes (I think you got this mixed up).

Reply:
Updated